features/step_definitions/helper_services.rb

```diff
@@ -183,7 +184,7 @@
       | port_spec | <%= cb.ldap_port %>:389 |
       })
     step %Q/the step should succeed/
-end
+end 
```

Comment:
Still has diff and one space. Suggest run git diff locally and check, before git push.

Reply:
removed space